bullet_train/docs/webhooks/incoming.md

```diff
@@ -5,3 +5,127 @@ Bullet Train makes it trivial to scaffold new endpoints where external systems c
 ```
 rails generate super_scaffold:incoming_webhooks
 ```
+
+## Security
+
+When receiving webhook events you need to ensure that they come from a trusted source. If the platform signs their webhook events, you can usually verify the event authenticity via their verification method. Bullet Train has webhook event signatures built in now. 
+
+### Managing the shared secret
+
+In order to verify signatures both sides need to have a shared secret. In the app that will be sending webhooks you'll set up an endpoint (via the UI or API) that points to the URL of the app that will be receiving them. After the endpoint is created you'll get a `webhook_secret` value. You should copy that value and set it as an `ENV` variable in the app that will be receiving the webhooks.
+
+For purposes of this example we'll assume that the shared secret is accessible to the receiving app as `ENV["BULLET_TRAIN_WEBHOOK_SECRET"]`.
+
+### Bullet Train apps receiving webhooks from another Bullet Train app
+
+In this case you can make direct use of the `BulletTrain::OutgoingWebhooks::Signature` class that comes with your Bullet Train installation.
+
+In the app that's going to be receiving the webhooks you'd do something like this:
+
+```
+rails generate super_scaffold:incoming_webhooks OtherBulletTrainApp
+```
+
+Then open `app/controllers/webhooks/incoming/other_bullet_train_app_webhooks_controller.rb` and add a `before_action :verify_authenticity` call that looks like this:
+
+```ruby
+before_action :verify_authenticity, only: [:create]
+
+def verify_authenticity
+  unless BulletTrain::OutgoingWebhooks::Signature.verify_request(request, ENV["BULLET_TRAIN_WEBHOOK_SECRET"])
+    # Consider alerts here and responding with a status and error message that
+    # don't reveal that there is an endpoint at all.
```

Comment:
I don't entirely understand what this comment is trying to tell me. Is it suggesting that I should do something other than what's happening below? Or is it describing why we're doing this particular thing below?